<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SyrahMail</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>80</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>935</x>
      <y>80</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>211</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>80</y>
      <width>221</width>
      <height>421</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>8</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive Manually</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="WriteMail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>125</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New Folder T</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New Mail T</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert DB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>371</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="MailInfo"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSyrahMail">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuWhatever">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionWhatever_2"/>
     <addaction name="actionWhatever_3"/>
    </widget>
    <addaction name="actionWrite"/>
    <addaction name="separator"/>
    <addaction name="menuWhatever"/>
    <addaction name="actionSetup"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAccounts">
    <property name="title">
     <string>Accounts</string>
    </property>
    <addaction name="actionAdd_mail_account"/>
    <addaction name="actionChange_default_account"/>
   </widget>
   <addaction name="menuSyrahMail"/>
   <addaction name="menuAccounts"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionWrite">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </action>
  <action name="actionWhatever_2">
   <property name="text">
    <string>whatever</string>
   </property>
  </action>
  <action name="actionWhatever_3">
   <property name="text">
    <string>whatever</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="text">
    <string>Add account</string>
   </property>
  </action>
  <action name="actionAdd_new_account">
   <property name="text">
    <string>Add new account</string>
   </property>
  </action>
  <action name="actionAdd_mail_account">
   <property name="text">
    <string>Add mail account</string>
   </property>
  </action>
  <action name="actionChange_default_account">
   <property name="text">
    <string>Change default account</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionWrite</sender>
   <signal>triggered()</signal>
   <receiver>WriteMail</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
